File: app/src/main/java/br/com/danielamaral/fiapbandas/business/java/RegrasBusiness.java
package br.com.danielamaral.fiapbandas.business.java;

public class RegrasBusiness {

    public double getNotaBanda(String banda) throws Exception {
        if (banda.toLowerCase().equals("metallica")) {
            return 10.0;
        } else if (banda.toLowerCase().equals("bad religion")) {
            return 9.0;
        } else if (banda.toLowerCase().equals("callipso")) {
            return 2.5;
        } else {
            return 0.0;
        }
    }
}

File: app/src/main/java/br/com/danielamaral/fiapbandas/business/RegrasBusiness.kt
package br.com.danielamaral.fiapbandas.business

class RegrasBusiness {

    fun getNotaBanda(nomeBanda: String): Double {
        var notaBanda: Double = 0.0;

        if (nomeBanda.toLowerCase() == "metallica") {
            notaBanda = 10.0
        } else if (nomeBanda.toLowerCase() == "bad religion") {
            notaBanda = 9.0
        } else if (nomeBanda.toLowerCase() == "callipso") {
            notaBanda = 2.0
        }

        return notaBanda;
    }
}